src/coffee.js

```diff
@@ -14,17 +14,19 @@ const coffee = [
   {
     name: "Aynu",
     profile: "clean",
-    acidity: "round",
+    characteristic: "round",
```

Comment:
Don't name these keys back please. Keys must be called `acidity` for this (clean, round, or acidic), `characteristics` for taste/tasing profile, and `roastLevels` for roast levels (instead of roast profile). Values for `roastLevels` must be `"dark"` etc. instead of `"dark roast"` etc. Please don't change these things.